index.html

```diff
@@ -358,6 +433,33 @@ <h3 class="faq__title">FAQ</h3>
 <script src="js/AgdaGlobalScripts.js"></script>
 <script src="js/grayscale.js"></script>
 
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+<script type="text/javascript" src="http://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+<script type="text/javascript"
+        src="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.pack.min.js"></script>
+<script type="text/javascript">
```

Comment:
if possible, please move this code to a js file instead of it being written inside the html code